### PR TITLE
Release/0.1

### PR DESCRIPTION
**Pull request by Wokarol**

### CODE REVIEW

File: foo.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Aaaaaaaaa
```
Comment (lucasstarsz) [inline foo.txt:1]: Very useful

File: useless.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This is... useless!
+
+Even more useless actually...
+
+More...
```
Comment (lucasstarsz) [inline useless.txt:5]: More? What more!??!?

File: foo.txt (hunk 1 above)
Comment (Wokarol) [reply foo.txt:1]: Yes

**Review by lucasstarsz — APPROVED** [review]
Yeah, go on then

**Pull request merged**